src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { TodoCondition } from '../../types/TodoCondition';
+
+type Props = {
+  todo: Todo,
+  todoCondition: TodoCondition,
+  onDeleteTodo?: (todoId: number) => void,
+  toggleTodo?: (curentTodo: Todo, status?: boolean | undefined) => void,
+  handleSubmitEditing?: (id: number, thisTitle: string) => void,
+};
+
+export const TodoItem: React.FC<Props> = ({
+  todo,
+  todoCondition,
+  onDeleteTodo = () => { },
+  toggleTodo = () => { },
+  handleSubmitEditing = () => { },
```

Comment:
```suggestion
  onDeleteTodo = () => {},
  toggleTodo = () => {},
  handleSubmitEditing = () => {},
```

Reply:
also you can create function `noop` as util or constants and pass in all places
`const noop = () => {}`